content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -614,12 +656,10 @@ jwt:
   {{% /tab %}}
   {{< /tabs >}}
 
-Importantly, the API server is not an OAuth2 client, rather it can only be
-configured to trust a single issuer. This allows the use of public providers,
-such as Google, without trusting credentials issued to third parties. Admins who
-wish to utilize multiple OAuth clients should explore providers which support the
-`azp` (authorized party) claim, a mechanism for allowing one client to issue
-tokens on behalf of another.
+##### Limitations
+
+1. Distributed claims do not work via [CEL](/docs/reference/using-api/cel/) expressions.
+1. Egress selector configuration is not supported for calls to `issuer.url` and `issuer.discoveryURL`.
 
 Kubernetes does not provide an OpenID Connect Identity Provider.
 You can use an existing public OpenID Connect Identity Provider (such as Google, or
```

Comment:
The `Support OpenID connect discovery; not all do` bit below doesn't actually match the flexibility that `discoveryURL` provides?

Reply:
I've updated this. LMK what you think!